Dockerfile

```diff
@@ -187,6 +187,7 @@ ENV _APP_SERVER=swoole \
     _APP_EXECUTOR_SECRET=a-random-secret \
     _APP_EXECUTOR_HOST=http://appwrite-executor/v1 \
     _APP_EXECUTOR_RUNTIME_NETWORK=appwrite_runtimes \
```

Comment:
I think we are missing:
-  value in `.env`
- binding in `docker-compose.yml`
- Both of the above for released files (they are in PHTMLs)

Reply:
As you can see my comment in the description, we are not yet documenting this Environment variable publicly, for now we will be using it internally.